src/raiden_libs/cli.py

```diff
@@ -102,12 +119,27 @@ def decorator(func: Callable) -> Callable:
                     default=False,
                     help="Enable or disable logging in JSON format",
                 ),
+                click.option(
+                    "--flamegraph",
+                    help="Directory in which to save a stack profile to produce flame graphs.",
```

Comment:
Does this hurt performance? If so, should we hide or mention that it's not recommended?

Reply:
It surely does, although I don't know how much. But option help texts are not made for long explanations, so I would not add it.